cli/command/container/opts.go

```diff
@@ -12,7 +12,9 @@ import (
 	"time"
 
 	"github.com/Sirupsen/logrus"
+	"github.com/docker/cli/cli/compose/loader"
```

Comment:
I agree , I was thinking about that too. Currently it outputs a compose type, so it would possibly require a larger change to create some shared type, and then translate it to a compose type later.

I think it's inevitable there is going to be sharing between compose and other cli components. I think we use `service` functions in `convert` right now